tests/backup/backup_scale_test.go

```diff
@@ -0,0 +1,211 @@
+				}
+			}
+			wg.Wait()
+		})
+		Step("Delete all Backup locations from px-admin", func() {
+			log.InfoD("Delete Backup locations from px-admin")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "failed to fetch ctx for admin")
+			AllBackupLocationMap, err := getAllBackupLocations(ctx)
+			log.FailOnError(err, "Fetching all backup locations")
+			for backupLocationUID, backupLocationName := range AllBackupLocationMap {
+				wg.Add(1)
+				go func(backupLocationName, backupLocationUID string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					err := DeleteBackupLocationWithContext(backupLocationName, backupLocationUID, orgID, true, ctx)
+					Inst().Dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying deletion of backup location [%s]", backupLocationName))
+				}(backupLocationName, backupLocationUID)
+			}
+			wg.Wait()
+		})
+		Step("Wait for Backup location deletion", func() {
+			log.InfoD("Wait for Backup location deletion")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "failed to fetch ctx for admin")
+			AllBackupLocationMap, err := getAllBackupLocations(ctx)
+			log.FailOnError(err, "Fetching all backup locations")
+			for backupLocationUID, backupLocationName := range AllBackupLocationMap {
+				wg.Add(1)
+				go func(backupLocationName, backupLocationUID string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					err := Inst().Backup.WaitForBackupLocationDeletion(ctx, backupLocationName, backupLocationUID, orgID, backupLocationDeleteTimeout, backupLocationDeleteRetryTime)
+					Inst().Dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying waiting for backup location [%s] deletion", backupLocationName))
+				}(backupLocationName, backupLocationUID)
+			}
+			wg.Wait()
+		})
+	})
+	JustAfterEach(func() {
+		defer func() {
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+			EndPxBackupTorpedoTest(scheduledAppContexts)
+		}()
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		log.InfoD("Deleting the restores")
+		for _, restoreName := range restoreNames {
+			wg.Add(1)
+			go func(restoreName string) {
+				defer GinkgoRecover()
+				defer wg.Done()
+				err = DeleteRestore(restoreName, orgID, ctx)
+				dash.VerifySafely(err, nil, fmt.Sprintf("Deleting restore [%s]", restoreName))
+			}(restoreName)
+		}
+		wg.Wait()
+		log.InfoD("Deleting the deployed apps after the testcase")
+		opts := make(map[string]bool)
+		opts[SkipClusterScopedObjects] = true
+		DestroyApps(scheduledAppContexts, opts)
+		log.InfoD("Deleting the px-backup objects")
+		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
+		log.InfoD("Switching context to destination cluster for clean up")
+		err = SetDestinationKubeConfig()
+		log.FailOnError(err, "Unable to switch context to destination cluster [%s]", destinationClusterName)
+		DestroyApps(scheduledAppContexts, opts)
```

Comment:
I think this would not work because of custom namespace mapping.

Reply:
Right, removed the code for now.